content/01-motivation.md

```diff
@@ -1,32 +1,20 @@
----
-layout: episode
-title: "Motivation"
-teaching: 10
-exercises: 0
-questions:
-  - "Why should we write tests?"
-objectives:
-  - "Discuss the advantages of writing and maintaining tests in research software."
-keypoints:
-  - "Tests make sure that expected functionality is preserved"
-  - "Tests help both users and developers of your code"
-  - "Tests help managing complexity"
```

Comment:
Should these be transferred into the main body as admonitions (see example page).  Or is it integrated in text naturally?

Reply:
I find we never really discuss it anyway and these are text section titles, so "built in".

Reply:
I will try to build these in ...

Reply:
This is a good point.  I guess carpentries sees it as important.  I can see the benefit of making some things explicit, but it can be just as much in main body - but should be made clear

Reply:
Perhaps we can better clarify our example episodes to say what needs to be said in each section, to make this clear to the learners.  Next week...